scripts/health_check.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+from enum import Enum
+
+import requests
+
+
+ENV_ENUM = Enum(
+    'ENV',
+    [
+        ('dev', 'https://addons-dev.allizom.org'),
+        ('stage', 'https://addons.allizom.org'),
+        ('prod', 'https://addons.mozilla.org'),
+        # TODO: maybe we could use the local environmnet here
+        ('test', ''),
+    ],
+)
+
+
+class Fetcher:
+    def __init__(self, env: ENV_ENUM, verbose: bool = False):
+        self.environment = ENV_ENUM[env]
+        self.verbose = verbose
+
+    def _fetch(self, path: str) -> dict[str, str] | None:
+        url = f'{self.environment.value}/{path}'
+        if self.verbose:
+            print(f'Requesting {url} for {self.environment.name}')
+
+        try:
+            response = requests.get(url)
+            response.raise_for_status()
+            try:
+                data = response.json()
+            except json.JSONDecodeError as e:
+                if self.verbose:
+                    print(f'Error decoding JSON for {url}: {e}')
+
+        except requests.exceptions.HTTPError as e:
+            if self.verbose:
+                print(f'Error fetching {url}: {e}')
+
+        if self.verbose and data is not None:
+            print(json.dumps(data, indent=2))
+
+        return data
+
+    def version(self):
+        if self.environment.name == 'test':
+            return {}
+        return self._fetch('__version__')
+
+    def monitors(self):
+        if self.environment.name == 'test':
+            return {
+                'up': {'state': True},
+                'down': {'state': False, 'status': 'something is wrong'},
+            }
+        return self._fetch('services/monitor.json')
```

Comment:
We likely want to check `__heartbeat__` and `services/__heartbeat__` (separately) instead

Reply:
why?

Reply:
Because both matter (and `monitor.json` is just an alias for `services/__heartbeat__` - doesn't matter which URL you use, but if adding a check for `__heartbeat__` might as well be consistent and use `services/__heartbeat__` for the other one)

Reply:
I see, they have different checks. That wasn't super clear until I looked closely at the views. At the risk of continuing that confusion. I've added a __healthcheck__ endpoint which checks all of the relevant monitors in a single go. I think it's worth the trade off since we don't want to disturb any existing callers to the original endpoints and this way our healtcheck does precisely what we want in a single request. wdyt?

Reply:
Yeah. The trick with these is that `__heartbeat__` is really mandatory for addons-server to function, where as `services/__heartbeat__` is just services, if they are down the site would still work.

Reply:
One issue with adding a new endpoint: dev/stage/prod nginx's config needs to be updated, (and frontend's list of URLs valid without an app/locale, though it's purely for cleanliness here, as we are not going to call that from the frontend).

If you kept just calling both separately you wouldn't have to deal with that.

Reply:
Reverted to using the existing endpoints.